plugins/inputs/execd/execd_test.go

```diff
@@ -0,0 +1,90 @@
+// +build !windows
+
+package execd
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf/agent"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/models"
+
+	"github.com/influxdata/telegraf/plugins/parsers"
+
+	"github.com/influxdata/telegraf"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExternalInputWorks(t *testing.T) {
+	jsonParser, err := parsers.NewInfluxParser()
+	assert.NoError(t, err)
```

Comment:
Recommend using `require.NoError(t, err)` form almost always.  Otherwise the test will continue and that usually just makes debugging the test more confusing.